### PR TITLE
bump Android Support Library to 28.0.0

### PR DESCRIPTION
**Pull request by dulmandakh**
bump Android Support Library to 28.0.0

Changelog:
----------
[Android] [Changed] - bump Android Support Library to 28.0.0

Test Plan:
----------
CI is green, and everything should work as usual

### CODE REVIEW

**Comment by hramos** [comment]
Looks like CI is failing on test_android.

**Comment by dulmandakh** [comment]
@hramos please import this PR.

File: ReactAndroid/src/main/third-party/android/support/v4/BUCK [hunk 1]
```diff
@@ -1,116 +1,10 @@
 load("//tools/build_defs:fb_native_wrapper.bzl", "fb_native")
+load("//tools/build_defs/oss:rn_defs.bzl", "react_native_dep")
 
 fb_native.android_library(
     name = "lib-support-v4",
     visibility = ["PUBLIC"],
     exported_deps = [
-        ":android-lifecycle-common",
```
Comment (mdvacca) [inline ReactAndroid/src/main/third-party/android/support/v4/BUCK]: Can you revert changes in this file?

File: ReactAndroid/src/main/third-party/android/support/BUCK [hunk 2]
```diff
@@ -24,3 +47,667 @@ fb_native.genrule(
     bash = "ls",
     visibility = ["PUBLIC"],
 )
+
```
Comment (mdvacca) [inline ReactAndroid/src/main/third-party/android/support/BUCK]: Can you revert changes in this file?

File: ReactAndroid/src/main/third-party/android/support/BUCK [hunk 3]
```diff
@@ -1,19 +1,42 @@
 load("//tools/build_defs:fb_native_wrapper.bzl", "fb_native")
-load("//tools/build_defs/oss:rn_defs.bzl", "react_native_dep", "rn_android_library")
+load("//tools/build_defs/oss:rn_defs.bzl", "react_native_dep", "rn_android_library", "rn_prebuilt_jar")
```
Comment (mdvacca) [inline ReactAndroid/src/main/third-party/android/support/BUCK]: Can you revert changes in this file?

File: ReactAndroid/src/main/third-party/android/support-annotations/BUCK [hunk 4]
```diff
@@ -1,14 +1,10 @@
 load("//tools/build_defs:fb_native_wrapper.bzl", "fb_native")
-load("//tools/build_defs/oss:rn_defs.bzl", "rn_prebuilt_jar")
+load("//tools/build_defs/oss:rn_defs.bzl", "react_native_dep")
```
Comment (mdvacca) [inline ReactAndroid/src/main/third-party/android/support-annotations/BUCK]: Can you revert changes in this file and upgrade version of support library from 27.1.1 to 28.0.0 in the original file?

File: ReactAndroid/src/main/third-party/android/support/v7/appcompat-orig/BUCK [hunk 5]
```diff
@@ -19,8 +19,7 @@ rn_android_library(
         ":res-for-appcompat",
     ],
     exported_deps = [
-        ":classes-for-react-native",
-        react_native_dep("third-party/android/support-annotations:android-support-annotations"),
+        react_native_dep("third-party/android/support:appcompat-v7"),
```
Comment (mdvacca) [inline ReactAndroid/src/main/third-party/android/support/v7/appcompat-orig/BUCK]: Can you revert changes in this file?

File: ReactAndroid/src/main/third-party/android/support/v4/BUCK (hunk 1 above)
Comment (mdvacca) [reply ReactAndroid/src/main/third-party/android/support/v4/BUCK]: And upgrade dependencies from 27.1.1 to 28.0.0 in the original file?

**Comment by dulmandakh** [comment]
@mdvacca yep, it seems that there is no choice but do everything old way. So I'll create another PR to bump Android Support Library to 28.0.0.

**Pull request closed without merging**